### PR TITLE
UI: Adds tidy_revoked_certs and revoked_cert_deleted_count to PKI tidy status page 

### PR DESCRIPTION
**Pull request by hellobontempo**
Fixes duplicate `tidy_revocation_queue`  which should have been `tidy_revoked_certs` 

Screenshot 2023-09-21 at 12 40 32 PM

### CODE REVIEW

File: ui/lib/pki/addon/components/page/pki-tidy-status.ts
Comment (hellobontempo) [inline ui/lib/pki/addon/components/page/pki-tidy-status.ts:1]: Organized params a bit so it's easier to follow what goes where

File: ui/lib/pki/addon/components/page/pki-tidy-status.ts [hunk 1]
```diff
@@ -57,24 +60,26 @@ export default class PkiTidyStatusComponent extends Component<Args> {
   tidyStatusGeneralFields = [
     'time_started',
     'time_finished',
-    'last_auto_tidy_finished',
```
Comment (hellobontempo) [inline ui/lib/pki/addon/components/page/pki-tidy-status.ts:60]: This param doesn't behave as expected. It behaves similarly to time_finished, and isn't specific to auto-tidying

File: ui/lib/pki/addon/components/page/pki-tidy-status.ts [hunk 2]
```diff
@@ -57,24 +60,26 @@ export default class PkiTidyStatusComponent extends Component<Args> {
...
     'cert_store_deleted_count',
+    'revoked_cert_deleted_count',
     'missing_issuer_cert_count',
-    'revocation_queue_deleted_count',
   ];
 
   tidyStatusConfigFields = [
     'tidy_cert_store',
-    'tidy_revocation_queue',
```
Comment (hellobontempo) [inline ui/lib/pki/addon/components/page/pki-tidy-status.ts:68]: duplicate (enterprise only, below) - this should have originally been `tidy_revoked_certs`

File: ui/lib/pki/addon/components/page/pki-tidy-status.ts [hunk 3]
```diff
@@ -57,24 +60,26 @@ export default class PkiTidyStatusComponent extends Component<Args> {
...
-    'tidy_cross_cluster_revoked_certs',
```
Comment (hellobontempo) [inline ui/lib/pki/addon/components/page/pki-tidy-status.ts:69]: enterprise only (moved below)

File: ui/lib/pki/addon/components/page/pki-tidy-status.ts [hunk 4]
```diff
@@ -57,24 +60,26 @@ export default class PkiTidyStatusComponent extends Component<Args> {
   tidyStatusGeneralFields = [
     'time_started',
     'time_finished',
-    'last_auto_tidy_finished',
     'cert_store_deleted_count',
+    'revoked_cert_deleted_count',
     'missing_issuer_cert_count',
-    'revocation_queue_deleted_count',
```
Comment (hellobontempo) [inline ui/lib/pki/addon/components/page/pki-tidy-status.ts:63]: enterprise only moved below to `crossClusterOperation` array

**Review by zofskeez — APPROVED** [review]
LGTM 👍

**Pull request merged**